### PR TITLE
(#42) - More OO loggers

### PR DESCRIPTION
**Pull request by victornoel**
This is for #42.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#80](https://github.com/g4s8/ghman/issues/80) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#80](https://github.com/g4s8/ghman/issues/80) is assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CJT4R32Q3/6ddac548-2066-43b4-8e6a-1a4bd9907403); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/main/java/com/g4s8/ghman/web/TkGitHubAuthorization.java [hunk 1]
```diff
@@ -60,6 +58,12 @@
      */
     private final Func<org.takes.Request, String> auth;
 
+    /**
+     * Logger.
+     */
+    @SuppressWarnings("PMD.LoggerIsNotStaticFinal")
+    private final Logger logger;
```
Comment (olenagerasimova) [inline src/main/java/com/g4s8/ghman/web/TkGitHubAuthorization.java]: @victornoel logger -> log ([Don't Create Objects That End With -ER](https://www.yegor256.com/2015/03/09/objects-end-with-er.html))

**Comment by victornoel** [comment]
@olenagerasimova I've pushed a new commit (and force-push/rebased because of gitlint)

**Comment by olenagerasimova** [comment]
@rultor merge

**Comment by 0crat** [comment]
@olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ammaratef45[/z](https://www.0crat.com/u/ammaratef45) please review this job completed by @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Code review was too long (9 days), architects (@g4s8) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
The job [#80](https://github.com/g4s8/ghman/issues/80) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ammaratef45** [comment]
@olenagerasimova You found less than three problems in the PR, please confirm you will try to find at least three problems next time

**Comment by olenagerasimova** [comment]
@ammaratef45 I confirm

**Comment by ammaratef45** [comment]
@olenagerasimova thanks

**Comment by ammaratef45** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Pull request merged**